core/storage/config/gae_models.py

```diff
@@ -61,3 +61,57 @@ def get_export_policy():
 
     def commit(self, committer_id, commit_cmds):
         super(ConfigPropertyModel, self).commit(committer_id, '', commit_cmds)
+
+
+class ConfigVariableSnapshotMetadataModel(
```

Comment:
I'm not sure I can comfortably support a "config variable" model when we have a "config property" model already. The two names are too similar and this is going to cause confusion. There also does not seem to be much information in the design doc about how config variables interact with config properties, or whether there's going to be a migration from one to the other, or ... anything about config properties at all, really.

So I think the naming for this needs to be more clearly delineated to be separate from the existing structures. Or, if they represent the same thing, then some plan needs to be in place so that we don't end up with two similarly-named things in the codebase at any given time.

Reply:
They are similar because in the planned future work of dynamic feature gating, the existing config properties can be replaced by config variables (but they co-exist for now). I was naming the storage and domain entities by ConfigVariables with the intent to make a generic base for that future work.
I can add detailed comments to explain this in code, and we can instead name everything by FeatureFlags-* and move them somewhere like 'core/storage/features/gae_models.py' to avoid any confusion, but in the future (out of the scope of this project) it may require more work to migrate.
What do you think about these two options?

Reply:
Update: Changed the naming to "platform parameter".